.circleci/config.yml

```diff
@@ -405,6 +405,28 @@ jobs:
             git commit -m "Auto-push due to change on main branch [ci skip]" \
               && git push \
               || echo "Skipping push since it looks like there were no changes"
+  sync-dags-repo:
+    executor: docker/machine
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "e4:30:50:41:53:f0:d6:3a:bb:c9:38:54:2d:ca:56:41"
+      - run:
+          name: 🤖 Update DAGs repository
+          command: |
+            ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+            git config --global user.email "dataops@mozilla.com"
+            git config --global user.name "CircleCI Job"
+            git clone --branch main git@github.com:mozilla/telemetry-airflow-dags.git
```

Comment:
Could we use `generated-sql` instead of `main` here?

Reply:
`mozilla/telemetry-airflow-dags` uses submodules in the `main` branch. Any specific reason why we would use a different branch?

Reply:
Actually, your comment made me realise we don't pull the right branch.

`git pull origin dags` is a copy-paste error from `telemetry-airflow` and should be `git pull origin generated-sql` instead.

Reply:
Yes, we don't use DAGs from bigquery-etl's main branch anymore. The setup is the following
![image](https://github.com/mozilla/bigquery-etl/assets/1668835/fc80fb64-a8e5-445a-9557-405bdab88a72)

where CI from three repos update their respective git submodules in `telemetry-airflow-dags` main branch.

Reply:
You can see the repo here https://github.com/mozilla/telemetry-airflow-dags which has been configured for telemetry-airflow and bigquery-etl. private-bigquery-etl is next